forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-gateway/forgerock-openbanking-uk-aspsp-rs-gateway-server/src/main/java/com/forgerock/openbanking/aspsp/rs/api/event/v3_1_2/eventsubscription/EventSubscriptionApiController.java

```diff
@@ -149,7 +154,7 @@ public EventSubscriptionApiController(RSEndpointWrapperService rsEndpointWrapper
                         (String tppId) -> {
                             HttpHeaders additionalHttpHeaders = new HttpHeaders();
                             additionalHttpHeaders.add(OpenBankingHttpHeaders.X_OB_CLIENT_ID, tppId);
-                            return rsStoreGateway.toRsStore(request, additionalHttpHeaders, Collections.emptyMap(), OBEventSubscriptionsResponse1.class, obEventSubscriptionsParam);
+                            return rsStoreGateway.toRsStore(request, additionalHttpHeaders, Collections.emptyMap(), OBEventSubscriptionResponse1.class, obEventSubscriptionsParam);
```

Comment:
I'm surprised this didn't break the corresponding integration test, as it fails for me!